### PR TITLE
[ADD] fleet: create new vehicles

### PR DESCRIPTION
**Pull request by larm-odoo**
Adding a new document and section to HR.  No fleet documentation currently exists.

### CODE REVIEW

**Comment by larm-odoo** [comment]
Waiting for some feedback/answers from Quentin so this doc can be finished- not ready for review yet.

**Comment by larm-odoo** [comment]
Hi @jcs-odoo  - can you help with the codeowner issue? This is a new doc for a new app (no previous doc existed for fleet).

**Comment by jcs-odoo** [comment]
> Hi @jcs-odoo - can you help with the codeowner issue? This is a new doc for a new app (no previous doc existed for fleet).

Hi @larm-odoo !

All good for the codeowner team

I haven't read the doc, but I saw the title "Add a vehicle". Usually, we try to avoid verbs in titles. --> https://www.odoo.com/documentation/master/contributing/documentation/content_guidelines.html#titles-and-headings

When we explain a record on a documentation page, it is not only to _add_ a new record from scratch (like we explain in video tutorials, for example), but mostly to describe how it works in general, as a reference (which can be both to create a new record, view or edit an existing one).

I hope this is helpful. Have a good day :)

**Comment by larm-odoo** [comment]
Thank you @jcs-odoo  - I had tagged you at the suggestion of a coworker when I got the original error, but I am glad it was a non-issue! As for the name, thank you for pointing that out, and I have updated it so it is in a better format =)

**Review by jero-odoo — CHANGES_REQUESTED** [review]
Nice work @larm-odoo, only a few notes. Let me know if you have any questions. Thanks!

File: content/applications/hr/fleet/new_vehicle.rst
Comment (larm-odoo) [reply content/applications/hr/fleet/new_vehicle.rst:10]: I didn't realize that's what we referred to it as =D  I note that so I don't so that again (I have a feeling it's in other docs).

File: content/applications/hr/fleet/new_vehicle.rst
Comment (larm-odoo) [reply content/applications/hr/fleet/new_vehicle.rst:32]: Thank you- I didn't know that!

File: content/applications/hr/fleet/new_vehicle.rst
Comment (larm-odoo) [reply content/applications/hr/fleet/new_vehicle.rst:47]: This was from Quentin: "The assignment date field is used to define when the car will be available, if it is empty it means the car is available."  It's an awful name for that field, because you are right- I would thin it was when it was assigned, but that's not true. I tried to make this make more sense in my description.

**Comment by larm-odoo** [comment]
Hi @tiku-odoo - this is set for the 2nd peer review!

File: content/applications/hr/fleet/new_vehicle.rst
Comment (tiku-odoo) [inline content/applications/hr/fleet/new_vehicle.rst:105]: Do any of these values impact accounting?

File: content/applications/hr/fleet/new_vehicle.rst
Comment (tiku-odoo) [inline content/applications/hr/fleet/new_vehicle.rst:135]: average?

**Review by tiku-odoo — COMMENTED** [review]
@larm-odoo 

Good start to this doc- I like the definitions you've compiled. However, I think this doc needs some intros to each section, so it isn't textbook. I've left a few comments on things to change. Notable-- it isn't the calendar module the dropdown/selector is populating. It is just a standard date selector. We should not be using the word "module." A module in Odoo is an application or set of Python instructions to be executed. All this does is select a date unless I am missing something. 

Creating a company deserves a warning admonition block. This causes subscriptions to go up $$$ and potentially shut a database down if they don't resolve the upsell.

A few other questions - Are there any links with the accounting app? I see a lot of data input here. 

Loop me back in for a quick review once you've made the changes. Thanks :+1: 
Tim

File: content/applications/hr/fleet/new_vehicle.rst
Comment (tiku-odoo) [inline content/applications/hr/fleet/new_vehicle.rst:141]: This model tab is a duplicate from your other doc--- 

does this info auto populate? b/c its going to be listed in two docs identical. I don't think we want to be repeating ourselves

Lines 142-152
some of lines 178-194

**Review by tiku-odoo — COMMENTED** [review]
@larm-odoo -- Can you explain the copied info?

File: content/applications/hr/fleet/new_vehicle.rst
Comment (larm-odoo) [reply content/applications/hr/fleet/new_vehicle.rst:13]: I see what you mean, but I think it's best to keep the docs for the specific versions, to keep things clear. I do the version specific changes, and I kind of like keeping everything very organized like that. I'll leave this for @StraubCreative to decide since it could go either way.

File: content/applications/hr/fleet/new_vehicle.rst
Comment (larm-odoo) [reply content/applications/hr/fleet/new_vehicle.rst:47]: Thanks for the explanation that a module is for python- I didn't realize I was using the incorrect term for that!

File: content/applications/hr/fleet/new_vehicle.rst
Comment (larm-odoo) [reply content/applications/hr/fleet/new_vehicle.rst:95]: good catch- not sure how I missed that!

File: content/applications/hr/fleet/new_vehicle.rst
Comment (larm-odoo) [reply content/applications/hr/fleet/new_vehicle.rst:135]: Thank you, yes, and it's from the manufacturer/dealership- not a changeable number. I added that as well.

File: content/applications/hr/fleet/new_vehicle.rst
Comment (larm-odoo) [reply content/applications/hr/fleet/new_vehicle.rst:105]: I assume so- but these are no "suggested" numbers- they should be concrete- meaning, they do not have any say in what these numbers are. The values are based on formulas (the residual value is) but I am not sure the exact formula. I'll add a note to check with accounting for more info.

File: content/applications/hr/fleet/new_vehicle.rst
Comment (larm-odoo) [reply content/applications/hr/fleet/new_vehicle.rst:31]: This is just the driver section - I saw there were sections at the top, versus the tabs below, so I added "section" so it's more clear.

File: content/applications/hr/fleet/new_vehicle.rst
Comment (larm-odoo) [reply content/applications/hr/fleet/new_vehicle.rst:58]: added section here too and a sentence

File: content/applications/hr/fleet/new_vehicle.rst
Comment (larm-odoo) [reply content/applications/hr/fleet/new_vehicle.rst:63]: OMG, it took me so long to see the second "s". I checked in 15, 15.2, to see what I got wrong, I thought it just said Chassis and not "number". =D Thanks for catching this!

File: content/applications/hr/fleet/new_vehicle.rst
Comment (larm-odoo) [reply content/applications/hr/fleet/new_vehicle.rst:141]: Hi Tim- this is the same info because it's the same process. I thought it was good to have the info in both docs since a person might go to one doc for something, and another for another reason- and then, in both docs, they would have everything they were looking for. This is something I will also let @StraubCreative decide, since I can see your pint (I can link to another doc) but I also like having everything they need here.

**Comment by larm-odoo** [comment]
Hi @tiku-odoo - I addressed the things I could, and replied to a few things I wasn't 100% on. Thanks for the module explanation- that has been fixed. I didn't see any specific accounting documents I could link to- nothing really mentions vehicles or anything related to it. I left the 2 biggest open-ended/unresolved comments for Zac to chime in on. If you want to take a second look, it's ready!

File: content/applications/hr/fleet/new_vehicle.rst
Comment (tiku-odoo) [inline content/applications/hr/fleet/new_vehicle.rst:135]: Will this information populate after selecting the model type? This is documented already in your other doc, we should be anchoring to that doc if that's the case. https://github.com/odoo/documentation/pull/5763

**Review by tiku-odoo — COMMENTED** [review]
Looks good Lara- still wondering about the same information in your two fleet PRs-- does one populate the other? 

I think we need some clarification on whether it should populate from the model one sets up in https://github.com/odoo/documentation/pull/5763 (line 149)-- if not okay, I'm just wondering why we're instructing the user to input the same information twice. 

https://github.com/odoo/documentation/pull/5763
`Model
*****

- :guilabel:`Seats Number`: Enter how many passengers the vehicle can accommodate.
- :guilabel:`Doors Number`: Enter the number of doors the vehicle has.
- :guilabel:`Color`: Enter the color of the vehicle.
- :guilabel:`Model Year`: Enter the year the vehicle was manufactured.
- :guilabel:`Trailer Hitch`: Check this box if the vehicle has a trailer hitch installed.
- 
- Engine
******

- :guilabel:`Fuel Type`: Select the type of fuel the vehicle uses form the drop-down menu. The
  options are :guilabel:`Diesel`, :guilabel:`Gasoline`, :guilabel:`Hybrid Diesel`, :guilabel:`Hybrid
  Gasoline`, :guilabel:`Plug-in Hybrid Diesel`, :guilabel:`Plug-in Hybrid Gasoline`,
  :guilabel:`CNG`, :guilabel:`LPG`, :guilabel:`Hydrogen`, or :guilabel:`Electric`.
- :guilabel:`CO2 Emissions`: Enter the average carbon dioxide emissions the vehicle produces in
  grams per kilometer (g/km). This information is provided by the car manufacturer.
- :guilabel:`CO2 Standard`: Enter the standard amount of carbon dioxide in grams per kilometer
  (g/km) for a similar sized vehicle.
- :guilabel:`Transmission`: Select the transmission type from the drop-down menu, either
  :guilabel:`Manual` or :guilabel:`Automatic`.
- :guilabel:`Power`: If the vehicle is electric or hybrid, enter the power the vehicle uses in
  kilowatts (kW).
- :guilabel:`Horsepower`: Enter the vehicle's horsepower in this field.
- :guilabel:`Horsepower Taxation`: Enter the amount that is taxed based on the size of the vehicles
  engine. This is determined by local taxes and regulations, and varies depending on the location.
  It is recommended to check with the accounting department to ensure this value is correct.
- :guilabel:`Tax Deduction`: This field auto-populates according to the engine specifications, and
  cannot be modified. The percentage is based on the localization settings and local tax laws.`

https://github.com/odoo/documentation/pull/5759

`Model
~~~~~
- :guilabel:`Model Year`: Enter the year the vehicle was manufactured.
- :guilabel:`Transmission`: Select the transmission type from the drop-down menu, either
  :guilabel:`Manual` or :guilabel:`Automatic`.
- :guilabel:`Color`: Enter the color of the vehicle.
- :guilabel:`Seats Number`: Enter how many passengers the vehicle can accommodate.
- :guilabel:`Doors Number`: Enter the number of doors the vehicle has.
- :guilabel:`Trailer Hitch`: Check this box if the vehicle has a trailer hitch.

Engine
~~~~~~

- :guilabel:`Horsepower`: Enter the vehicle's horsepower in this field.
- :guilabel:`Power`: If the vehicle is electric or hybrid, enter the power the vehicle uses in
  kilowatts (kW).
- :guilabel:`Fuel Type`: Select the type of fuel the vehicle uses form the drop-down menu. The
  options are :guilabel:`Diesel`, :guilabel:`Gasoline`, :guilabel:`Hybrid Diesel`, :guilabel:`Hybrid
  Gasoline`, :guilabel:`Plug-in Hybrid Diesel`, :guilabel:`Plug-in Hybrid Gasoline`,
  :guilabel:`CNG`, :guilabel:`LPG`, :guilabel:`Hydrogen`, or :guilabel:`Electric`.
- :guilabel:`CO2 Emissions`: Enter the average carbon dioxide emissions the vehicle produces in
  grams per kilometer (g/km). This information should be available from the car manufacturer, or
  from the dealership the car was purchased from.
- :guilabel:`Co2 Standard`: Enter the standard amount of carbon dioxide for a similar sized vehicle.

.. image:: new_vehicle/new-vehicle-model.png
   :align: center
   :alt: The new vehicle form, showing the vehicle tax section.`

File: content/applications/hr/fleet/new_vehicle.rst
Comment (larm-odoo) [reply content/applications/hr/fleet/new_vehicle.rst:135]: OMG TIM- NOW I understand!!! Because you CAN enter the info when making a new model, and if you select the new saved model it DOES auto-populate! I was thinking of this form as in terms of making a new car, everything new- and a lot of the info they ask for is identical. BUT if they select a model that is already in the system, then they will have a lot of the fields auto-populated. OK, now I know which should be the main one and which should direct to the other doc. Thank you- that took me a while to wrap my head around!

**Comment by larm-odoo** [comment]
Hi Tim- I am adding an anchor to a document that is NOT published yet- so there will be an error. I am going to work with Sam on this, to see if there is an easier/better way to handle this, since I don't know if I'll be bale to test this myself with unpublished docs.  I have fixed the issue though in theory- now I just have to get it to work =). I'll tag you for a re-review when it's ready but wanted to let you know the "a-ha" moment I had!

File: content/applications/hr/fleet/new_vehicle.rst
Comment (jero-odoo) [inline content/applications/hr/fleet/new_vehicle.rst:24]:
```suggestion
- :guilabel:`Model`: select the vehicle's model from the drop-down menu. If the model is not listed,
  type in the model name and click either :guilabel:`Create` or :guilabel:`Create and Edit`.
- :guilabel:`License Plate`: enter the vehicle's license plate number in this field.
- :guilabel:`Tags`: select any tags from the drop-down menu, or type in a new tag. There is no limit
  on the amount of tags that can be selected.
```

File: content/applications/hr/fleet/new_vehicle.rst
Comment (jero-odoo) [inline content/applications/hr/fleet/new_vehicle.rst:38]:
```suggestion
- :guilabel:`Driver`: select the driver from the drop-down menu, or type in a new driver and click
```

File: content/applications/hr/fleet/new_vehicle.rst
Comment (jero-odoo) [inline content/applications/hr/fleet/new_vehicle.rst:52]:
```suggestion
- :guilabel:`Mobility Card`: if the selected driver has a mobility card listed on their employee
  card in the *Employees* application, the mobility card number will appear in this field. If there
  is no mobility card listed and one should be added, :doc:`edit the employee card
  </applications/hr/employees/new_employee>` in the *Employees* application.
- :guilabel:`Future Driver`: if the next driver for the vehicle is known, select the next driver
  from the drop-down menu, or type in the next driver and click either :guilabel:`Create` or
  :guilabel:`Create and Edit`.
- :guilabel:`Plan To Change Car`: if the current driver set for this vehicle plans to change their
  vehicle, either because they are waiting on a new vehicle that is being ordered, or this is a
  temporary vehicle assignment and they know which vehicle they will be driving next, check this
  box. If the current driver does not plan to change their vehicle and use this current vehicle, do
  not check this box.
- :guilabel:`Assignment Date`: select the date the vehicle will be available for another driver
```

File: content/applications/hr/fleet/new_vehicle.rst
Comment (jero-odoo) [inline content/applications/hr/fleet/new_vehicle.rst:60]:
```suggestion
   A driver does **not** have to be an employee, but a driver must be listed in the *Contacts*
```
I think anytime we are trying to emphasize a word (like not, must, cannot, etc) bold is better than italics.

File: content/applications/hr/fleet/new_vehicle.rst
Comment (jero-odoo) [inline content/applications/hr/fleet/new_vehicle.rst:84]:
```suggestion
- :guilabel:`Immatriculation Date`: select the date the vehicle is acquired using the drop-down
  calendar.
- :guilabel:`Cancellation Date`: select the date the vehicle lease will expire, or when the vehicle
  will be no longer available, using the drop-down calendar.
- :guilabel:`Chassis Number`: enter the chassis number in the field. This is known in some countries
  as the :abbr:`VIN (Vehicle Identification Number)` number.
- :guilabel:`Last Odometer`: enter the last known odometer reading in the number field. Using the
  drop-down menu next to the number field, select whether the odometer reading is in kilometers
  :guilabel:`(km)` or miles :guilabel:`(mi)`.
- :guilabel:`Fleet Manager`: select the fleet manager from the drop-down menu, or type in a new
  fleet manager and click either :guilabel:`Create` or :guilabel:`Create and Edit`.
- :guilabel:`Location`: type in the location for the vehicle in the field. The most common scenario
  for when this field would be populated is if a company has several office locations. The typical
  office location where the vehicle is located would be the location entered.
- :guilabel:`Company`: select the company that the vehicle will be used for and associated with from
```

File: content/applications/hr/fleet/new_vehicle.rst
Comment (jero-odoo) [inline content/applications/hr/fleet/new_vehicle.rst:110]:
```suggestion
- :guilabel:`Horsepower Taxation`: enter the amount that is taxed based on the size of the vehicles
  engine. This is determined by local taxes and regulations, and varies depending on the location.
  It is recommended to check with the accounting department to ensure this value is correct.
- :guilabel:`Disallowed Expense Rate`: this is the amount of non-deductible expenses for the
  vehicle. This amount is not counted towards any deductions on a tax return or as an allowable
  expense when calculating taxable income. It is recommended to check with the accounting department
  to ensure the value(s) entered are correct.

  - :guilabel:`Start Date`: enter the :guilabel:`Start Date` and :guilabel:`(%) Percentage` for when
```

File: content/applications/hr/fleet/new_vehicle.rst
Comment (jero-odoo) [inline content/applications/hr/fleet/new_vehicle.rst:126]:
```suggestion
- :guilabel:`First Contract Date`: select the start date for the vehicle's first contract using
  the drop-down calendar. Typically this is the day the vehicle is purchased or leased.
- :guilabel:`Catalog Value (VAT Incl.)`: enter the MSRP (Manufacturer's Suggested Retail Price) for
  the vehicle at the time of purchase or lease.
- :guilabel:`Purchase Value`: enter the purchase price or the value of the lease for the vehicle.
- :guilabel:`Residual Value`: enter the current value of the vehicle.
```

**Review by jero-odoo — APPROVED** [review]
great job @larm-odoo . Had a few additional notes, let me know if you have any questions. Thanks!

File: content/applications/hr/fleet/new_vehicle.rst
Comment (samueljlieber) [inline content/applications/hr/fleet/new_vehicle.rst:116]:
```suggestion
    XX.XX format. Repeat this for all entries needed.
```

**Review by samueljlieber — APPROVED** [review]
Nice job @larm-odoo! This doc looks great, I only have two small corrections that I will push up after this review, please see below. Approving now, thank you!

**Comment by samueljlieber** [comment]
Implemented technical changes in https://github.com/odoo/documentation/commit/de5d3825c6fdf14b56691d5c8be79f7528d60be7.

@StraubCreative this PR is good to go 👍 and can be FWP all the way up to `master`. 
Just to note, we _may_ have a conflict on the FWP for 16.0 due to the addition of a custom anchor on new_employee.rst in this PR.

**Review by StraubCreative — APPROVED** [review]
@robodoo r+

**Comment by StraubCreative** [comment]
@robodoo retry

**Pull request closed without merging**
